### PR TITLE
[DOCS] Update absolute 2023.1 links to 2023.2, fix references to RH 8.7->8.8, fix references of python 3.6->3.7, add accurracy tuning note

### PR DESCRIPTION
**Pull request by dkalinowski**

### CODE REVIEW

File: client/python/ovmsclient/lib/README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ cd model_server/client/python/ovmsclient/lib
 
 ## Installation
 
-**Note:** The client library requires Python in version >= 3.6.
+**Note:** The client library requires Python in version >= 3.7.
```
Comment (mzegla) [inline client/python/ovmsclient/lib/README.md:18]: How is that? Why 3.6 is no longer valid?

File: docs/performance_tuning.md [hunk 2]
```diff
@@ -179,11 +179,20 @@ docker run --rm -d -v ${PWD}/models/public/resnet-50-tf:/opt/model -p 9001:9001
...
       **Note:** When launching benchmark app use `-inference_only=false`. Otherwise OV avoids setting input tensor of inference each time which is not comparable flow to OVMS.
-2. [Launch OVMS benchmark client](https://docs.openvino.ai/2023.1/ovms_demo_benchmark_client.html) on the same machine as OVMS
-3. [Launch OVMS benchmark client](https://docs.openvino.ai/2023.1/ovms_demo_benchmark_client.html) from remote machine
+2. [Launch OVMS benchmark client](https://docs.openvino.ai/2023.2/ovms_demo_benchmark_client.html) on the same machine as OVMS
+3. [Launch OVMS benchmark client](https://docs.openvino.ai/2023.2/ovms_demo_benchmark_client.html) from remote machine
 4. Measure achievable network bandwidth with tools such as [iperf](https://github.com/esnet/iperf)
+
+## Analyzing accuracy issues
+
+It is recommended to compare accuracy results versus benchmark app.
```
Comment (dtrawins) [inline docs/performance_tuning.md:193]:
```suggestion
Note that the target devices GPU and NVIDIA are changing by default the model execution precision from FT32 to FT16.
It is recommended to compare accuracy with the original precision.
```

File: docs/performance_tuning.md [hunk 3]
```diff
@@ -179,11 +179,20 @@ docker run --rm -d -v ${PWD}/models/public/resnet-50-tf:/opt/model -p 9001:9001
...
+
+1. Launch OV benchmark app with **the same** plugin parameters as when starting OVMS container *--plugin_config*.
```
Comment (dtrawins) [inline docs/performance_tuning.md:195]:
```suggestion
It is possible to enforce a specific runtime precision by using a plugin config parameter `INFERENCE_PRECISION_HINT`. For example:

```

File: docs/performance_tuning.md [hunk 4]
```diff
@@ -179,11 +179,20 @@ docker run --rm -d -v ${PWD}/models/public/resnet-50-tf:/opt/model -p 9001:9001
...
+2. Confirm that in both cases you are using the same `INFERENCE_PRECISION_HINT` parameter which by default may optimize inner layers by changing its precision.
```
Comment (dtrawins) [inline docs/performance_tuning.md:196]: `--plugin_config '{"INFERENCE_PRECISION_HINT": "f32"}'` or `--plugin_config '{"INFERENCE_PRECISION_HINT": "f16"}'`

File: client/python/ovmsclient/lib/README.md (hunk 1 above)
Comment (dkalinowski) [reply client/python/ovmsclient/lib/README.md:18]: @dtrawins

**Review by dtrawins — APPROVED** [review]

**Review by mzegla — APPROVED** [review]

File: docs/performance_tuning.md (hunk 2 above)
Comment (dkalinowski) [reply docs/performance_tuning.md]: Changed

File: docs/performance_tuning.md (hunk 3 above)
Comment (dkalinowski) [reply docs/performance_tuning.md]: Changed

File: docs/performance_tuning.md (hunk 4 above)
Comment (dkalinowski) [reply docs/performance_tuning.md]: Changed

**Pull request merged**